src/AbstractLiveSets/AbstractLiveSets.jl

```diff
@@ -0,0 +1,22 @@
+module AbstractLiveSets
+
+import Unitful: unit
+
+using ..AbstractWalkers
+using ..EnergyEval
+using ..Potentials
+using ..Hamiltonians
+
+export AbstractLiveSet
+export AtomWalkers, LatticeWalkers
+export LJAtomWalkers, LatticeGasWalkers
+
+abstract type AbstractLiveSet end
+
+# Base.eltype(::AbstractLiveSet{T}) where {T} = T
```

Comment:
@yangmr04, is this a comment or commented-out code? If it's the latter, should we remove it?

Reply:
Yes, it's the latter. Shall be removed.